app/containers/MediaCallHeader/MediaCallHeader.test.tsx

```
@@ -94,8 +96,30 @@ describe('MediaCallHeader', () => {
+		// Drive the FSM into awaitingMediaCall via the real event path (not stubs).
+		// MediaCallHeader reads store.call; during the pre-bind window call is still null,
+		// so the header must render the empty placeholder — not a partial/broken UI.
+		(callLifecycle as any)._resetForTesting();
+		useCallStore.setState({ call: null });
+
+		// Wire native adapter → FSM.
+		const native = new InMemoryVoipNative();
+		callLifecycle.attach(native);
+		await native.attach({ onEvent: callLifecycle.handleNativeEvent.bind(callLifecycle) });
+
+		await act(async () => {
```

ESLint

[error] Async arrow function has no 'await' expression.

(require-await)